src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/PaymentSuccess.cshtml

```diff
@@ -12,14 +12,23 @@
     <div class="govuk-!-padding-bottom-5"></div>
     <div class="govuk-panel govuk-panel--confirmation govuk-!-width-two-thirds">
         <h1 class="govuk-panel__title">
-            You are now registered as a small producer
+            You are now registered as a small producer for @Model.ComplianceYear
         </h1>
         <div class="govuk-panel__body">
             Payment reference number:<br><strong>@Model.PaymentReference</strong>
         </div>
     </div>
     <p class="govuk-body">You have been emailed a receipt to the email address provided during the payment.</p>
     <div class="govuk-!-width-two-thirds">
+        <h2 class="govuk-heading-m">Payment summary</h2>
+        <table id="payment-summary-table" class="govuk-table">
+            <tbody class="govuk-table__body">
+                <tr class="govuk-table__row">
```

Comment:
@stevelee-civica  just want to check that Wave is ok with this html?

Reply:
no errors from wave